examples/experimental/dagster-airlift/README.md

```diff
@@ -1,26 +1,174 @@
+
+To add definitions that an Airlift-enabled deployment will observed, you need to use the `orchestrated_defs` argument to `build_defs_from_airflow_instance`
+
+_Note: This also accepts an object of type `Definitions`. We have recently added a function, `Definitions.merge`, which allows users to combine and compose `Definitions` objects, which this utilizes._
+
+For most projects we anticipate the following process:
+
+- Create a new factory class for every operator type or specialized use case of an operator (e.g. a BashOperator invoking a dbt project).
+- Use that factory to create definitions associating with each Airflow task.
+
+In our example we have a pre-written factory class (in `examples/experimental/dagster-airlift/dagster_airlift/dbt/multi_asset.py`) that is associated with invoking a dbt project:
+
+```python
+@dataclass
+class DbtProjectDefs(DefsFactory):
+    dbt_project_path: Path
+    name: str
+    group: Optional[str] = None
+
+    def build_defs(self) -> Definitions:
+        dbt_manifest_path = self.dbt_project_path / "target" / "manifest.json"
+
+        @dbt_assets(manifest=json.loads(dbt_manifest_path.read_text()), name=self.name)
+        def _dbt_asset(context: AssetExecutionContext, dbt: DbtCliResource):
+            yield from dbt.cli(["build"], context=context).stream()
+
+        if self.group:
+            _dbt_asset = _dbt_asset.with_attributes(
+                group_names_by_key={key: self.group for key in _dbt_asset.keys}
+            )
+
+        return Definitions(
+            assets=[_dbt_asset],
+            resources={
+                "dbt": DbtCliResource(
+                    project_dir=self.dbt_project_path, profiles_dir=self.dbt_project_path
+                )
+            },
+        )
+```
+
+We imagine most people will have to customize this, so we encourage you to copy and paste it for now.
+
+Then you can add instances of the Factory to your `build_defs_from_airflow_instance` call:
+
+```python
+
+defs = build_defs_from_airflow_instance(
+    airflow_instance=airflow_instance,
+    orchestrated_defs=defs_from_factories(
+        DbtProjectDefs(
+            name="dbt_dag__build_dbt_models",
+            dbt_project_path=dbt_project_path(),
+            group="dbt",
+        ),
+    ),
+)
+```
+
+### Mapping assets to tasks
+
+We default to a "convention over configuration" approach to affiliate Dagster assets with Airflow tasks.
+
+Note the naming convention `dbt_dag__build_dbt_models`. By default we use this convention to encode the dag name (`dbt_dag`) and task id (`build_dbt_models`) of the Airflow task actually orchestrating the computation.
+
+This name gets parsed and set corresponding tags (`airlift/dag_id` and `airlift/task_id`). Alternatively you can set those tags explicitly if you don’t want to rely on the naming convention.
```

Comment:
Maybe worth noting that these tags are set on both the asset and the underlying nodedef, for people who know what that is.

Reply:
Also noting that you have to specify the tags under op_tags on an asset.